pykechain/models/activity.py

```diff
@@ -244,6 +245,13 @@ def edit(self, name=None, description=None, start_date=None, due_date=None, assi
             else:
                 raise TypeError('Assignee should be a string')
 
+        if status:
+            if isinstance(status, (str, text_type)):
+                update_dict.update({'status': status})
+                self.status = status
```

Comment:
we do not know if the request went through (APIError and stuff) so this is preliminary here